content/cookies.md

```diff
@@ -19,8 +19,8 @@ Find out more about [how to manage cookies](http://www.aboutcookies.org).
 We use Google Analytics software to collect information about how you use our website. We do this to help make sure the website is meeting the needs of its users and to help us make improvements.
 Google Analytics stores information about:
 
-- the pages you visit on the Pension Wise website  - how long you spend on each page 
-- how you got to the website  
+- the pages you visit on the Pension Wise website - how long you spend on each page 
```

Comment:
This feels like a mistake. looks like `how long you spend on each page` should be on it's own line.